godot-codegen/src/util.rs

```diff
@@ -268,6 +298,23 @@ fn to_rust_type_uncached(ty: &str, ctx: &mut Context) -> RustTy {
         }
     }
 
+    if ty.ends_with('*') {
+        // Pointer type; strip '*', see if const, and then resolve the
+        // inner type.
+        let mut ty = ty.to_owned();
+        ty.pop();
```

Comment:
Instead of allocation + `pop`, you could maybe use a slice.

If this makes subsequent code harder, feel free to ignore.

Reply:
:+1: